features/calc.yml

```diff
@@ -3,5 +3,11 @@ description: The `calc()` CSS function computes mathematical expressions such a
 spec: https://drafts.csswg.org/css-values-3/#calc-notation
 group: css
 caniuse: calc
+status:
+  compute_from: css.types.calc
 compat_features:
   - css.types.calc
+  - css.types.calc.color_component
```

Comment:
This one actually belongs in `relative-colors`, but is tagged in BCD incorrectly. I think remove it from here.